src/services/cloudwatchevent/index.ts

```diff
@@ -145,3 +153,5 @@ export const producedEventsSupportedServices = [
 export const producedDeployOutputTypes = [];
 
 export const consumedDeployOutputTypes = [];
+
+export const supportsTagging = true;
```

Comment:
I think the Handel docs don't show that the CloudWatch Events service supports Tags (the rules don't, but we do tag the stack here), can we add that to the docs as part of this change?